### ISSUE TITLE
End Time is not displayed in Start Time's timezone

### ISSUE THREAD

**Issue by cliffordp** [Xtendify/Simple-Calendar#259]
Example event: 
![image](https://cloud.githubusercontent.com/assets/1812179/23474164/5f530778-fe78-11e6-85d8-269c10a9d28b.png)

Calendar is set to use "Event source default"

The 7pm should display as 4pm, like this:
`March 22, 2017 @ 12:00 pm - 4:00 pm`

Or timezone abbreviations should be displayed, like this:
`March 22, 2017 @ 12:00 pm PDT - 7:00 pm EDT`

**Comment by nickyoung87** [comment]
@cliffordp Timezone stuff is weird when it comes to the calendar and Google. Not sure if this is the issue here but there are several ways the timezone can get out of sync with the Google calendar and the plugin. Here is a doc showing the differences: https://docs.simplecalendar.io/timezone-settings/

Are you setting the timezone the same in all places?

**Comment by cliffordp** [comment]
@nickyoung87 the situation here is that Google Calendar allows creating an event (not editing an existing event) with different timezones for start time and end time... and the plugin fails to account for this situation.

**Comment by nickyoung87** [comment]
Are you using "event source" in the calendar settings? It should be accounting for that if I recall correctly, but I may be wrong.

**Comment by cliffordp** [comment]
Yes, I am using that setting. Please attempt to replicate the issue.
